ui-lib/src/main/java/co/electriccoin/zcash/ui/common/repository/AddressBookRepository.kt

```diff
@@ -1,22 +1,48 @@
+@file:Suppress("DEPRECATION")
```

Comment:
Which part of the code causes the deprecation warning?

Reply:
GoogleSignIn

Reply:
Could we have a follow-up ticket for the deprecated component replacement?